### ISSUE TITLE
Hi, About the radar graph in your paper

### ISSUE THREAD

**Issue by HypnosXC** [muzairkhattak/multimodal-prompt-learning#5]
Sorry to Bother. I am working on a similar work and I saw the radar graph of yours . It is good to display the result of experiment and I am wondering how to draw such a graph. Could you please tell the API or soft for drawing such a graph? Thank you!

**Comment by muzairkhattak** [comment]
Hi @HypnosXC, 

Thank you for showing interest in our work.

The radar plot in our **MaPLe** paper is drawn using a python code. We have uploaded a jupyter notebook at this [link](https://drive.google.com/drive/folders/1usjZqy9BJ878qbeFlZcsqyKFzLyFAo60?usp=sharing) that can replicate the same radar plot which we have used in our paper. 

Kindly look into the notebook and use it accordingly. We hope it will be helpful. 

Thank you.
